### PR TITLE
UI visual overhaul

### PR DESCRIPTION
**Pull request by Dangercato**

### CODE REVIEW

File: Source/Private/Render/Windows/MainWindow.cpp [hunk 1]
```diff
@@ -32,28 +32,38 @@ bool MainWindow::IsEnabled()
...
-    ImGui::Begin("Kyber", &m_isEnabled, ImGuiWindowFlags_AlwaysAutoResize);
-    ImGui::Checkbox("Show Menu [INSERT]", &m_isEnabled);
-    ImGui::Checkbox("Show Credits", &m_creditsWindow->m_isEnabled);
-    ImGui::Separator();
+    ImGui::Begin("KYBER", &m_isEnabled, ImGuiWindowFlags_AlwaysAutoResize);
+
     if (!g_program->m_server->m_running)
     {
-        if (ImGui::Button("Start a Server"))
+        if (ImGui::Button("SERVER SETUP"))
```
Comment (BattleDash) [inline Source/Private/Render/Windows/MainWindow.cpp:39]: If we're going to combine "Start a Server" and "Server Settings" then there's no point for this if statement, it's running the same code in both clauses.

File: Source/Private/Render/Windows/MainWindow.cpp [hunk 2]
```diff
@@ -32,28 +32,38 @@ bool MainWindow::IsEnabled()
...
         }
     }
-    if (ImGui::Button("Join a Server"))
+    if (ImGui::Button("SERVER BROWSER"))
     {
         m_clientWindow->m_isEnabled = true;
     }
+    ImGui::Separator();
+
+    // ImGui::Text("Toggle Menu: [INSERT]");
```
Comment (BattleDash) [inline Source/Private/Render/Windows/MainWindow.cpp:57]: Code comments like this are ugly.

File: Source/Private/Render/Windows/ServerWindow.cpp [hunk 3]
```diff
@@ -41,12 +41,12 @@ bool DrawScoreboardPlayer(std::vector<ServerPlayer*> playerList, int index)
 
 void ServerWindow::Draw()
 {
-    ImGui::Begin("Server Settings", &m_isEnabled, ImGuiWindowFlags_AlwaysAutoResize);
-    GameSettings* gameSettings = Settings<GameSettings>("Game");
-    ImGui::Text("Game Mode:");
+    ImGui::Begin("SERVER SETTINGS", &m_isEnabled, ImGuiWindowFlags_AlwaysAutoResize);
+    GameSettings* gameSettings = Settings<GameSettings>("GAME");
```
Comment (BattleDash) [inline Source/Private/Render/Windows/ServerWindow.cpp:45]: Do not uppercase SettingsManager objects. This needs to be ("Game").

File: Source/Private/Render/Windows/ServerWindow.cpp [hunk 4]
```diff
@@ -133,20 +133,21 @@ void ServerWindow::Draw()
...
-        ImGui::Checkbox("Auto Balance Players", &Settings<WSGameSettings>("Whiteshark")->AutoBalanceTeamsOnNeutral);
+        ImGui::Text("AI SETTINGS");
+        ImGui::SliderInt("AI COUNT", &aiSettings->ForcedServerAutoPlayerCount, -1, 64);
+        ImGui::Checkbox("UPDATE AI", &aiSettings->UpdateAI);
+        ImGui::SameLine();
+        ImGui::Checkbox("AI IGNORE PLAYERS", &aiSettings->ServerPlayersIgnoreClientPlayers);
+        ImGui::Checkbox("AUTO BALANCE TEAMS", &Settings<WSGameSettings>("Whiteshark")->AutoBalanceTeamsOnNeutral);
         ImGui::Separator();
-        ImGui::Text("Player List:");
+        ImGui::Text("PLAYER LIST");
```
Comment (BattleDash) [inline Source/Private/Render/Windows/ServerWindow.cpp:150]: If we're going to remove the colon I'd like to have this text be centered.

File: Source/Private/Utilities/ErrorUtils.cpp [hunk 5]
```diff
@@ -8,18 +8,18 @@ namespace Kyber
 {
 void ErrorUtils::ThrowException(LPCSTR message)
 {
-    LPCSTR base = "Something went wrong with Kyber. Please restart your game and report this to Kyber Staff if it "
+    LPCSTR base = "Something went wrong with KYBER. Please restart your game and report this on the KYBER Discord if it "
```
Comment (BattleDash) [inline Source/Private/Utilities/ErrorUtils.cpp:11]: Uppercasing Kyber here just looks silly.

File: Source/Private/Render/Windows/ServerWindow.cpp (hunk 4 above)
Comment (Dangercato) [reply Source/Private/Render/Windows/ServerWindow.cpp:150]: I tried this but it looked kinda jank with everything else being left-aligned

File: Source/Private/Render/Windows/MainWindow.cpp (hunk 1 above)
Comment (BattleDash) [reply Source/Private/Render/Windows/MainWindow.cpp:39]: Not fixed, you need to remove the top-level server running check.

**Pull request merged**